PocketKit/Sources/PocketKit/Article/ReadableViewModel/ReadableViewModel.swift

```diff
@@ -32,6 +32,7 @@ protocol ReadableViewModel: ReadableViewControllerDelegate {
     func showWebReader()
     func fetchDetailsIfNeeded()
     func externalActions(for url: URL) -> [ItemAction]
+    func clearPresentedWebReaderURL()
```

Comment:
why not add `clearIsPresentingReaderSettings` or `clearSharedActivity`

Reply:
I don't think it was strictly required in the contexts where we need to clear those things (since we would be dealing with concrete types instead of this protocol). But it might make sense to add them here anyway for consistency's sake.